pyproject.toml

```diff
@@ -76,14 +76,17 @@ profile = "black"
 [tool.pyright]
 include = [
   "ansible_ai_connect/ai/api/aws/wca_secret_manager.py",
-  "ansible_ai_connect/ai/api/model_client/bam_client.py",
-  "ansible_ai_connect/ai/api/model_client/dummy_client.py",
-  "ansible_ai_connect/ai/api/model_client/grpc_client.py",
-  "ansible_ai_connect/ai/api/model_client/http_client.py",
-  "ansible_ai_connect/ai/api/model_client/langchain.py",
-  "ansible_ai_connect/ai/api/model_client/llamacpp_client.py",
-  "ansible_ai_connect/ai/api/model_client/ollama_client.py",
-  "ansible_ai_connect/ai/api/model_client/wca_client.py",
+  "ansible_ai_connect/ai/api/model_pipelines/bam/pipelines.py",
+  "ansible_ai_connect/ai/api/model_pipelines/dummy/pipelines.py",
+  "ansible_ai_connect/ai/api/model_pipelines/grpc/pipelines.py",
```

Comment:
I think we can just remove this directory. We don't use it anymore.

Reply:
:boom: It is gone.